lib/generators/matestack_page/templates/matestack_page.erb

```diff
@@ -0,0 +1,26 @@
+class Pages::<% unless @namespace.nil? %><%= @namespace.camelize %>::<% end %><% unless @app_name.nil? %><%= @app_name.camelize %>::<% end %><%= class_name %> < Matestack::Ui::Page
+
+# OPTIONAL: Prepare data for the response method
+# def prepare
+#   API calls and db queries go here
+# end
+
+# MANDATORY: This is where the matestack magic happens
+def response
+  components {
+    plain 'Hello, mate!'
+  }
+end
+
+# OPTIONAL: Partials come after the response
```

Comment:
the partial is not beeing used in this generated page. I think this is missleading

Reply:
Will be removed